locales/settings/zh-TW.json

```
@@ -0,0 +1,195 @@
+{
+  "about.changelog": "更新日誌",
+  "about.feedbackInfo": "{{appName}} ({{commitSha}}) 正處於開發早期階段。如果您有任何反饋或建議，請隨時在我們的 GitHub 上 <OpenIssueLink>提交問題</OpenIssueLink> <ExternalLinkIcon />。",
+  "about.iconLibrary": "所使用的圖標庫版權由 <IconLibraryLink /> <ExternalLinkIcon /> 所有，不得重新分發。",
+  "about.licenseInfo": "{{appName}} 是且將永遠是一個免費且開源的項目。它遵循 {{license}} 授權。",
+  "about.sidebar_title": "關於",
+  "about.socialMedia": "社群媒體",
+  "actions.actionName": "操作 {{number}}",
+  "actions.action_card.add": "新增",
+  "actions.action_card.all": "全部",
+  "actions.action_card.block_rules": "封鎖規則",
+  "actions.action_card.custom_filters": "自訂過濾條件",
+  "actions.action_card.enable_readability": "啟用可讀性",
```

```suggestion
  "actions.action_card.enable_readability": "啟用閱讀模式",
```
Firefox call their readability `閱讀模式`
https://support.mozilla.org/zh-TW/kb/firefox-reader-view-clutter-free-web-pages